### ISSUE TITLE
Fix php-jwt dependences

### ISSUE THREAD

**Issue by brutto** [google/google-api-php-client#722]
This depends on google/auth-v0.3 witch depends on php-jwt-v2.0.0, but php-jwt-v3.0.0 already exist and use in google/auth-master.

So when i use api client and php-jwt on composer i could not get it normally.

**Comment by bshaffer** [comment]
I believe this issue would be fixed by setting google/auth to version ~v0.3, and then you could manually include google/auth: dev-master in your project's composer.json

Is composer throwing an error? If so paste it here.

**Comment by brutto** [comment]
In my ```composer.json``` with these dependences: 
```
...
"firebase/php-jwt": "^3.0",
"google/apiclient": "v2.0.0-RC1"
...
```

output after ```> composer update``` command:
```
Problem 1
    - google/apiclient v2.0.0-RC1 requires google/auth v0.3 -> satisfiable by google/auth[v0.3].
    - google/apiclient v2.0.0-RC1 requires google/auth v0.3 -> satisfiable by google/auth[v0.3].
    - google/auth v0.3 requires firebase/php-jwt 2.0.0 -> no matching package found.
    - google/auth v0.3 requires firebase/php-jwt 2.0.0 -> no matching package found.
    - Installation request for google/apiclient v2.0.0-RC1 -> satisfiable by google/apiclient[v2.0.0-RC1].
```

```composer.json``` with these dependences: 
```...
"firebase/php-jwt": "^3.0",
"google/auth": "dev-master",
"google/apiclient": "v2.0.0-RC1"
...
```

output problem:
```
  Problem 1
    - google/apiclient v2.0.0-RC1 requires google/auth v0.3 -> no matching package found.
    - google/apiclient v2.0.0-RC1 requires google/auth v0.3 -> no matching package found.
    - Installation request for google/apiclient v2.0.0-RC1 -> satisfiable by google/apiclient[v2.0.0-RC1].
```

Or am i right that you suggest to fork current ```google/apiclient``` and change ```composer.json``` there with ```"google/auth": "~v0.3"``` additionally?

**Comment by bshaffer** [comment]
Yes, that would certainly work. See [this post](http://mnapoli.fr/overriding-dependencies-with-composer/) for a quick way to do this.

I'll be tagging a `v0.4` version and updating this library's `composer.json` shortly to fix this.

**Comment by bshaffer** [comment]
This should now be fixed by #733. Can you verify by updating to `google/apiclient: dev-master`?

**Comment by brutto** [comment]
yap int ok now, thanks
